card/mod/standard/set/all/links.rb

```diff
@@ -1,81 +1,93 @@
-format do
-  # link is called by web_link, card_link, and view_link
-  # (and is overridden in other formats)
-  def link_to text, pathish, _opts={}
-    href = interpret_pathish pathish
+RESOURCE_TYPE_REGEXP = /^([a-zA-Z][\-+\.a-zA-Z\d]*):/
 
+format :html do
+  def link_to text=nil, opts={}
```

Comment:
I buy the first argument, not so sure about the second. Two hashes seems cleaner to me because the path key has a completely different meaning compared to all other keys in the hash.